src/markdown-pages/docs/managers/mq-manager.md

```diff
@@ -0,0 +1,233 @@
+The following snippet shows the code that is required to instantiate an IBM MQ queue:
+
+```java
+@QueueManager()
+public IMessageQueueManager qmgr;
+
+@Queue(archive = "true", name = "GALASA.INPUT.QUEUE")
+public IMessageQueue queue;
+```
+
+You can just as simply instantiate multiple IBM MQ queues:
+
+```java
+@QueueManager()
+public IMessageQueueManager qmgr;
+
+@Queue(archive = "true", name = "GALASA.INPUT.QUEUE")
+public IMessageQueue queue;
+	
+@Queue(archive = "false", name = "GALASA.INPUT.QUEUE2")
+public IMessageQueue queue2;
+	
+@Queue(tag = "NEWQUEUE")
+public IMessageQueue queue3;
+```
+</details>
+
+<details><summary>Put messages to an IBM MQ queue</summary>
+
+Multiple message objects can be passed to the _sendMessage_ method. The following snippets show the code required to create a text or binary message to put on an IBM MQ queue: 
+
+```java
+TextMessage tm = qmgr.createTextMessage(testData);
+queue.sendMessage(tm);
+```
+```java
+TextMessage tm = qmgr.createTextMessage(testData);
+queue.sendMessage(tm,tm,tm,tm,tm,tm,tm,tm);
+```
+```java
+byte[] input = {41,01,33,76};
+BytesMessage m = qmgr.createBytesMessage(input);
+queue.sendMessage(m);
+```
+
+</details>
+
+<details><summary>Read messages from an IBM MQ queue</summary>
+
+The following snippet shows the code required to read a message from an existing IBM MQ queue:
+
+```java
+Message m = queue.getMessage();
+String response = m.getBody(String.class);
+```
+</details>
+
+<details><summary>Clear messages from an IBM MQ queue</summary>
+
+The following snippet shows the code required to clear messages from an IBM MQ queue:
+
+```java
+queue.clearQueue();
+```
+</details>
+
+## Configuration Properties
+
+The following are properties used to configure the MQ Manager:
+
+<details>
+<summary>Instance ID for the tag CPS Property</summary>
+
+| Property: | Instance ID for the tag CPS Property |
+| --------------------------------------- | :------------------------------------- |
+| Name: | mq.tag.[tag].instanceid |
+| Description: | The instance for the specified tag |
+| Required:  | Yes |
+| Default value: | None |
+| Valid values: |  |
+| Examples: | <code>mq.tag.[tag].instanceid=QUEUEMGR1</code> |
+
+This property is associated with the _@QueueManager_ annotation attribute `tag`. You can use the `tag` attribute to avoid hardcoding in your test code. Because the properties that are associated with the attribute are stored in the CPS, the same test can run against a different queue manager without changing the test code. 
+
+</details>
+
+<details>
+<summary>Queue name for the tag CPS Property</summary>
+
+| Property: | Queue name for the tag CPS Property |
+| --------------------------------------- | :------------------------------------- |
+| Name: | mq.server.[tag].queuename |
+| Description: | The queue name for the specified tag |
+| Required:  | Yes |
+| Default value: | None |
+| Valid values: |  |
+| Examples: | <code>mq.server.[tag].queuename=GALASA.INPUT.QUEUE</code> |
```

Comment:
is actually mq.queue.tag.<tag>.queuename

Reply:
thanks, updated